### DISCUSSION TITLE
Version 2.206 released

### DISCUSSION THREAD

**Discussion by psb1558** [psb1558/Junicode-font#265] [General]
A new release today. This one includes 140 ligatures for tone marks, e.g.
<img width="304" alt="image" src="https://github.com/psb1558/Junicode-font/assets/16215767/ac70e2c7-8662-41f5-9a13-969d6771db69">
There are several improvements in the `junicodevf` package, especially an increase in the number of size gradations the package can handle. Documentation for TeX users has been merged into the *Junicode Manual*; TeX users should now obtain Junicode through CTAN, especially if using the static font, or via the TeX Live distribution, which includes both the static and the variable font.

This brings me to the end (I hope) of a rather long to do list; I hope this will be the last release for a while.

**Comment by Doc73** [comment]
Thank you very much for all your efforts to improve this excellent font.
I hope you can enjoy your well-deserved rest and then resume development with renewed strength.
